jina/logging/base.py

```diff
@@ -136,14 +139,28 @@ def success(self, msg: str, **kwargs):
         if self.log_level <= LogVerbosity.SUCCESS:
             sys.stdout.write(f'W:{self.context}:{self._planify(msg)}')
 
+class SuccessLogger(Protocol):
```

Comment:
Thank you for the work here, but since this part is undergoing a big refactor, I would appreciate if these changes can be left out of the PR